action.yml

```diff
@@ -84,16 +84,18 @@ runs:
       run: echo "STEW_CI_ARGS=" >> $GITHUB_ENV
 
     - name: Target specific project by name
-      if: inputs.run-stew == 'true' && inputs.project-name != '' && inputs.project-name != '.'
+      if: inputs.run-stew == 'true' && inputs.project-name != '' && !startsWith(inputs.project-name, '.')
       shell: bash
       env:
         PROJECT_NAME: ${{ inputs.project-name }}
       run: echo "STEW_CI_ARGS=$PROJECT_NAME --exact-match" >> $GITHUB_ENV
 
     - name: Target the project at the root of the repository
```

Comment:
Can you adjust the text around this feature, since it's no longer about targeting the project at the root.